### PR TITLE
Added prototype of helper dispatcher

### PR DESCRIPTION
**Pull request by SilkovAlexander**

### CODE REVIEW

File: gosh-dispatcher/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+target/
```
Comment (awnion) [inline gosh-dispatcher/.gitignore:1]: EOL

File: git-remote-gosh/src/git_helper/mod.rs [hunk 2]
```diff
@@ -251,6 +260,7 @@ pub async fn run(config: Config, url: &str) -> anyhow::Result<()> {
             (Some("capabilities"), None, None) => helper.capabilities().await?,
             (Some("list"), None, None) => helper.list(false).await?,
             (Some("list"), Some("for-push"), None) => helper.list(true).await?,
+            (Some("repo_version"), None, None) => helper.print_repo_version().await?,
```
Comment (andrew-kurochkin) [inline git-remote-gosh/src/git_helper/mod.rs:263]: I think it's better to name gosh related commands with "gosh_" prefix

Comment (SilkovAlexander) [reply git-remote-gosh/src/git_helper/mod.rs]: Fixed

File: gosh-dispatcher/.gitignore (hunk 1 above)
Comment (SilkovAlexander) [reply gosh-dispatcher/.gitignore]: Fixed

**Review by andrew-kurochkin — DISMISSED** [review]

**Review by andrew-kurochkin — APPROVED** [review]

**Pull request merged**